htmlReport.go

```diff
@@ -124,13 +124,16 @@ func createReportExecutableFile(reportsDir, pluginsDir string) {
 		os.Remove(exTarget)
 	}
 	if runtime.GOOS == "windows" {
-		createBatFileToExecuteHtmlReport(exPath, exTarget)
+		createBatFileToExecuteHTMLReport(exPath, exTarget)
 	} else {
-		createSymlinkToHtmlReport(exPath, exTarget)
+		createSymlinkToHTMLReport(exPath, exTarget)
 	}
 }
 
-func createBatFileToExecuteHtmlReport(exPath, exTarget string) {
+func createBatFileToExecuteHTMLReport(exPath, exTarget string) {
+	if isSaveExecutionResultDisabled() {
```

Comment:
The check can be made at the start of `createReportExecutableFile` function itself.